### PR TITLE
Adds production ready seeds

### PR DESCRIPTION
**Pull request by SevaDer14**

### CODE REVIEW

File: app/serializers/section/show_serializer.rb [hunk 1]
```diff
@@ -6,6 +6,7 @@ def attributes(*args)
     hash[:buttons] = object.buttons if object.regular?
     hash[:description] = object.description unless object.carousel?
     hash[:cards] = object.cards if object.carousel?
+    hash[:image] = object.image if object.regular?
```
Comment (tochman) [inline app/serializers/section/show_serializer.rb:9]: I see no test for this? Don't we have serializer specs that would need to be updated?

**Review by tochman — CHANGES_REQUESTED** [review]

File: app/serializers/section/show_serializer.rb (hunk 1 above)
Comment (SevaDer14) [reply app/serializers/section/show_serializer.rb:9]: Specs updated

File: db/seeds.rb [hunk 2]
```diff
@@ -5,39 +5,165 @@
...
-)
-section_no_image = Section.create(
-  view_id: about_self_care_view.id,
-  header: 'Section no Image',
-  description: 'A Solid App is composed of functions that we call Components. Component names in Solid follow the Pascal naming convention in which the first letter of each word in a component name is capitalized. This name can then be used as tags within our JSX like',
+# ---------- CREATING SECTONS FOR ABOUT US VIEW -----------
+puts 'creating sections for about_us view'
+background_and_setup = Section.create(
+  header: 'Background and Set-up',
+  description: 'his section tells vistor about Community Health West London background and setup',
```
Comment (tochman) [inline db/seeds.rb:12]: Spelling

File: db/seeds.rb [hunk 3]
```diff
@@ -5,39 +5,165 @@
...
-  header: 'Section regular',
-  description: 'A Solid App is composed of functions that we call Components. Component names in Solid follow the Pascal naming convention in which the first letter of each word in a component name is capitalized. This name can then be used as tags within our JSX like',
-  variant: 0,
-  image: image
-)
-section_no_image = Section.create(
-  view_id: about_self_care_view.id,
-  header: 'Section no Image',
-  description: 'A Solid App is composed of functions that we call Components. Component names in Solid follow the Pascal naming convention in which the first letter of each word in a component name is capitalized. This name can then be used as tags within our JSX like',
+# ---------- CREATING SECTONS FOR ABOUT US VIEW -----------
```
Comment (tochman) [inline db/seeds.rb:8]: Before we start seeding we have to make sure that we clear all data.(I commented on this on Slack) I suggest something like

```ruby
[Section, Cta, View].each do | model |
  model.destroy_all
end
```

File: spec/serilizers/section/show_serializer_spec.rb [hunk 4]
```diff
@@ -15,26 +15,32 @@
     end
   end
 
-  context ':regular' do
+  context ':regular' do    
     let(:sections) { create_list(:regular, 2, buttons: [create(:button)]) }
+    let!(:image_1) {create(:image, section: sections.first)}
+    let!(:image_2) {create(:image, section: sections.second)}
```
Comment (tochman) [inline spec/serilizers/section/show_serializer_spec.rb:21]: please format your code

**Review by tochman — CHANGES_REQUESTED** [review]

File: spec/serilizers/section/show_serializer_spec.rb (hunk 4 above)
Comment (SevaDer14) [reply spec/serilizers/section/show_serializer_spec.rb]: done

File: db/seeds.rb (hunk 3 above)
Comment (SevaDer14) [reply db/seeds.rb:13]: added

File: db/seeds.rb (hunk 2 above)
Comment (SevaDer14) [reply db/seeds.rb]: fixed

**Review by tochman — APPROVED** [review]

**Pull request merged**